src/examples/scala/project01/pipeline.scala

```diff
@@ -0,0 +1,6 @@
+import values.greeting
+import values.answer
```

Comment:
As an example showing imports... since it's now pure Scala code, can I reference external libraries or only libraries that happen to be included inside the Loam JAR?